src/pybind/mgr/dashboard/frontend/src/app/shared/api/mgr-module.service.ts

```diff
@@ -14,6 +14,7 @@ import { NotificationType } from '../enum/notification-type.enum';
 })
 export class MgrModuleService {
   private url = 'api/mgr/module';
+  updateCompleted$ = new Subject<void>();
```

Comment:
```suggestion
  private updateCompleted$ = new Subject<void>();
  updateCompleted$ = this.updateCompletedSubject.asObservable();
```
This will make only the service to sue `next` not other components.